_test_image.sh

```diff
@@ -5,9 +5,17 @@ tmp_dir=$(mktemp -d)
 trap "rm -r \"$tmp_dir\"" EXIT
 echo "hello world" > "$tmp_dir"/test.txt
 for i in {1..100};do
-  echo "hello world" >> "$tmp_dir"/test.txt
+  echo "hello world$i" >> "$tmp_dir"/test$i.txt
+  for j in {1..20};do
+    setfattr -n user.name$j -v value${i}_$j "$tmp_dir"/test$i.txt
+  done
 done
-dd if=/dev/zero of=test.ext4.tmp count=1024 bs=1024
-mkfs.ext4 test.ext4.tmp -d "$tmp_dir"
-echo -n F > test.ext4
-cat test.ext4.tmp >> test.ext4
+rm -r test{32,64}.ext4{,.tmp}
```

Comment:
This line doesn't make sense to me, this script is only run if the files don't exist. And they are files, not a directory so `-r` does nothing.

Reply:
done